### DISCUSSION TITLE
More payment options

### DISCUSSION THREAD

**Discussion by lewis45454** [planetscale/discussion#216] [Feature Requests] [answered]
PlanetScale currently only allows card payments, and I would like to see support for other payment methods like Paypal. 

Please share your thoughts.

**Comment by fooforge** [comment, marked as answer]
Hi @lewis45454,

Thanks for the feedback!

We are tracking interest for other payment options internally and I have added this thread to the discussion. It's not a priority item for us at the moment, but if we start supporting alternative ways to pay, it will surely be mentioned [in our changelog](https://planetscale.com/changelog).
